### PR TITLE
Remove unused util `$.fn.focusNextInputField`

### LINKED ISSUE #9179: remove unused utils
### Problem

I noticed some unused functions in `openlibrary/plugins/openlibrary/js/utils.js` we can remove them.

### Reproducing the bug

1. Go to ...
2. Do ...

* Expected behavior:
* Actual behavior:

### Context

- Browser (Chrome, Safari, Firefox, etc):
- OS (Windows, Mac, etc):
- Logged in (Y/N): Y
- Environment (prod, dev, local): prod

### PR DESCRIPTION
**Pull request by RayBB**
Closes #9179

### Testing

Did a fulltext search of the code and these aren't used anywhere I can see but a second check would be good.
Removes one unused piece of code and adds a comment about why we want to keep the rest.

### CODE REVIEW

**Review by jimchamp — CHANGES_REQUESTED** [review]
Thanks @RayBB!  The changes to the `excerpts` template are causing errors on the book edit page.  Switching from string interpolation to string concatenation should fix things.

File: openlibrary/templates/books/edit/excerpts.html [hunk 1]
```diff
@@ -66,7 +66,7 @@
                     {{excerpt}}
                     <br />
                     <i>
-                        {{cond(pages, "From page " + pages + ".", "")}}
+                        {{pages ? `From page ${pages}.` : "";}}
```
[the remark this answers is not in the document]
Comment (RayBB) [reply openlibrary/templates/books/edit/excerpts.html]: @jimchamp good catch... it is fixed as you suggested!

**Comment by cdrini** [comment]
Howdy folks, I've been thinking about this one and I think it might be best for us to close this. Even if we remove the places where our jsdef code uses e.g. `cond`, it's _expected_ that jsdef codeblocks should have access to these python globals. Removing the JS implementations here will make our code brittle, especially since we don't really have strong tests to verify that our python/jsdef templates are valid.

**Comment by cdrini** [comment]
To clarify, here is what I mean by jsdef using these files:

https://github.com/internetarchive/openlibrary/blob/1860c2562819330bf4657f8ed13d2c3d0ea336e2/openlibrary/templates/subjects.html#L84-L91

The `cond` call here uses the window.cond definition when this template is converted to/called from the JS.

**Comment by RayBB** [comment]
@cdrini thanks for replying.

Should we also keep `focusNextInputField` ?

Do we need to keep the `window.cond = cond;` for jsdef to work?
Or would it be feasible to move these to a window.jsdef.cond so they're not taking up so much at the top of index.js?

**Comment by cdrini** [comment]
`focusNextInputField` looks entirely unused; that looks like we can drop :+1: We do need the `window.cond` exposure in order for jsdef to work atm; if we wanted to change that, we would need to modify the way that the jsdef converts the templates to JS to check a certain global eg a JSDEF global. That would be nice in some future! But a little complicated to achieve. We'd need to change the parser to basically alter all globals to pull them from somewhere else. Here: https://github.com/internetarchive/openlibrary/blob/b46ff5135473c003f191d804bdd3b07cfa2c37cb/openlibrary/plugins/upstream/jsdef.py

**Comment by RayBB** [comment]
@cdrini sounds good to me. I've pushed code with a comment about this and removing `focusNextInputField`

**Review by cdrini — APPROVED** [review]
Lgtm! Tested on testing (the only place I can think where this might apply is the "add one?" link for description, which focusses on the description box; still works!) + can't find any references to that method.

**Pull request merged**